### DISCUSSION TITLE
Pause orientation at stops

### DISCUSSION THREAD

**Discussion by silvain** [osmandapp/OsmAnd#17428] [Ideas]
I regularly use OSMAnd’s navigation system in my car. The car display is linked to my phone with Android Auto.
When the map is oriented in the direction of travel, the map starts to rotate back and forth at each stop — due to the recaclulation of direction based on position every second and GPS inaccuracies. It takes around 10 secs until it stops rotating and display a small circle instead of an arrow.
It would be nice if this recalculation of the direction of travel could be paused when the speed is lower than a certain threshold — or maybe recalculated at more spread-out intervals so as to still show direction when traveling on foot).

**Comment by sjvudp** [comment]
Maybe a generic freeze/unfreeze button would be helpful (even while driving).

**Comment by silvain** [comment]
Presently, this issue appears to be solved for me. (Android 15.5.4.247, OSMAnd+ 5.3.10)
I had not tested it for some time, as I normally just use standard North-South orientation (I leave it to my brain to do the rotations...)

**Reply by sjvudp** [reply]
Well an obvious solution (when no longer moving, or not moving significantly) would be to keep the last direction.
"Significantly" could be comparing the estimated inaccuracy of the position fix with the calculated movement:
If the movement is less than the inaccuracy, there probably was not movement actually.
